lib/internal/abort_controller.js

```diff
@@ -185,25 +191,69 @@ class AbortSignal extends EventTarget {
     return signal;
   }
 
+  /**
+   * @param {AbortSignal[]} signals
+   * @returns {AbortSignal}
+   */
+  static any(signals) {
+    emitExperimentalWarning('AbortSignal.any');
```

Comment:
I wouldn't worry about emitting a warning here.

Reply:
I am fine either way, but in previous comments I was asked to add it 🙂 
https://github.com/nodejs/node/pull/47821#issuecomment-1531808945
https://github.com/nodejs/node/pull/47821#issuecomment-1532122283

Reply:
I'm fine without a warning _when the API lands at whatwg/dom_ but not before - so if you need someone who feels strongly about having an experimental warning until then - I'm that person :)

Reply:
This is supposed to land Monday so as long as this lands after that I'm fine with it landing without a warning.